tests/test_operators.py

```diff
@@ -5,73 +5,120 @@
 from pina.operators import grad, div, laplacian
 
 
-def func_vec(x):
+def func_vector(x):
     return x**2
 
 
 def func_scalar(x):
-    print('X')
     x_ = x.extract(['x'])
     y_ = x.extract(['y'])
-    mu_ = x.extract(['mu'])
-    return x_**2 + y_**2 + mu_**3
+    z_ = x.extract(['z'])
+    return x_**2 + y_**2 + z_**2
 
 
-data = torch.rand((20, 3), requires_grad=True)
-inp = LabelTensor(data, ['x', 'y', 'mu'])
-labels = ['a', 'b', 'c']
-tensor_v = LabelTensor(func_vec(inp), labels)
-tensor_s = LabelTensor(func_scalar(inp).reshape(-1, 1), labels[0])
-
+inp = LabelTensor(torch.rand((20, 3), requires_grad=True), ['x', 'y', 'z'])
+tensor_v = LabelTensor(func_vector(inp), ['a', 'b', 'c'])
+tensor_s = LabelTensor(func_scalar(inp).reshape(-1, 1), ['a'])
 
 def test_grad_scalar_output():
     grad_tensor_s = grad(tensor_s, inp)
+    true_val = 2*inp
     assert grad_tensor_s.shape == inp.shape
     assert grad_tensor_s.labels == [
         f'd{tensor_s.labels[0]}d{i}' for i in inp.labels
     ]
+    assert all((grad_tensor_s - true_val == 0).flatten())
```

Comment:
Instead of all(tensor1-tensor2 == 0), I would use `torch.allclose` since is more numerically stable

Reply:
Sure, I will make this change wherever applicable in the code.